README.md

```diff
@@ -1,24 +1,95 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Capstone App
 
-Things you may want to cover:
+The Ruby on Rails capstone project is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a group, so that you can see how much money you spent and on what.
 
-* Ruby version
+We created a Ruby on Rails application that allows the user to:
 
-* System dependencies
+- register and log in, so that the data is private to them.
+- introduce new transactions associated with a group.
+- see the money spent on each category.
 
-* Configuration
+![screenshot](./cap.png)
+![screenshot](./cap1.png)
+![screenshot](./cap2.png)
 
-* Database creation
+Additional description about the project and its features.
 
-* Database initialization
+## Built With
+Ruby 
+Ruby on Rails
 
-* How to run the test suite
```

Comment:
- [ ] Tests are missing from the project. 


- Make sure the app has unit tests for all the most important components of the RoR application.

- Make sure the app has integration tests for all the most important components of the RoR application.

Reply:
are you sure that tests are missing? which tests ?